### PR TITLE
Use 'documentation' wording instead of 'readme'

### LINKED ISSUE #4729: ReadMe.md should be renamed to Documentation.md
This will help us consolidate the Package page with the nuspec language. Otherwise the difference between description and ReadMe is vague.

### PR DESCRIPTION
**Pull request by chenriksson**
Fixes #4729

### CODE REVIEW

File: src/NuGetGallery/Strings.resx [hunk 1]
```diff
@@ -571,19 +571,19 @@ For more information, please contact '{2}'.</value>
     <value>Conversion of Markdown to HTML failed with '{0}'.</value>
   </data>
   <data name="PreviewReadMe_ReadMeMissing" xml:space="preserve">
-    <value>There is no ReadMe available to preview.</value>
+    <value>There is no markdown documentation available to preview.</value>
```
Comment (skofman1) [inline src/NuGetGallery/Strings.resx]: at some places you use "markdown documentation" and in others "documentation markdown". Is this intentional?

Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx]: Markdown is meant to be capitalized right?
Example: https://guides.github.com/features/mastering-markdown/

File: src/NuGetGallery/Views/Packages/_ImportReadMe.cshtml [hunk 2]
```diff
@@ -50,22 +50,22 @@
...
-                        <input type="url" data-bind="value: Edit.ReadMe.SourceUrl" name="Edit.ReadMe.SourceUrl" class="form-control" id="ReadMeUrlInput" placeholder="https://raw.githubusercontent.com/" />
+                        <input type="url" data-bind="value: Edit.ReadMe.SourceUrl" name="Edit.ReadMe.SourceUrl" class="form-control" id="ReadMeUrlInput" placeholder="https://raw.githubusercontent.com/*.md" />
                         <label id="ReadMeURLLabel" class="input-group-btn"></label>
                     </div>,        
                    "readme-url", active: false)
 
                 @ReadMePanel(
                     @<div class="input-group">
-                        <input type="text" id="ReadMeFileText" class="form-control" value="Browse or Drop files to select a ReadMe.md file..." aria-label="Upload Your ReadMe.md" readonly />
+                        <input type="text" id="ReadMeFileText" class="form-control" value="Browse for a documentation markdown file..." aria-label="Upload your documentation markdown" readonly />
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/_ImportReadMe.cshtml]: "documentation markdown file" sounds awkward compared to "Markdown documentation file"

File: src/NuGetGallery/Strings.resx [hunk 3]
```diff
@@ -571,19 +571,19 @@ For more information, please contact '{2}'.</value>
...
-    <value>There is no ReadMe available to preview.</value>
+    <value>There is no markdown documentation available to preview.</value>
   </data>
   <data name="ReadMeInvalidSourceType" xml:space="preserve">
-    <value>'{0}' is not a valid ReadMe source type.</value>
+    <value>'{0}' is not a valid markdown documentation source type.</value>
   </data>
   <data name="ReadMeMaxLengthExceeded" xml:space="preserve">
-    <value>ReadMe file must be less than {0} bytes.</value>
+    <value>The markdown documentation file must be less than {0} bytes.</value>
```
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx]: uppercase

File: src/NuGetGallery/Strings.resx [hunk 4]
```diff
@@ -571,19 +571,19 @@ For more information, please contact '{2}'.</value>
     <value>Conversion of Markdown to HTML failed with '{0}'.</value>
   </data>
   <data name="PreviewReadMe_ReadMeMissing" xml:space="preserve">
-    <value>There is no ReadMe available to preview.</value>
+    <value>There is no markdown documentation available to preview.</value>
   </data>
   <data name="ReadMeInvalidSourceType" xml:space="preserve">
-    <value>'{0}' is not a valid ReadMe source type.</value>
+    <value>'{0}' is not a valid markdown documentation source type.</value>
```
Comment (joelverhagen) [inline src/NuGetGallery/Strings.resx]: uppercase

**Pull request merged**